src/main/java/controllers/GameCreationController.java

```diff
@@ -26,12 +26,17 @@ public GameCreationController(GameCreationInputBoundary inputBoundary) {
      * @param playersInfo The User requested Player names and if they are to be ComputerPlayers or not.
      * @return A GameCreationResponseModel detailing if a Game was created.
      */
-    public GameCreationResponseModel createGameResponse(ArrayList<PlayerCreationInformation> playersInfo) {
+    public boolean createGameResponse(ArrayList<PlayerCreationInformation> playersInfo) {
         HashMap<String, Boolean> requestInfo = new HashMap<>();
         for(PlayerCreationInformation info: playersInfo) {
             requestInfo.put(info.getPlayerName(), info.getIsComputerPlayer());
         }
+        // immediately return false if there are duplicate names
+        if(requestInfo.keySet().size() != playersInfo.size()) {
+            return false;
+        }
         GameCreationRequestModel request = new GameCreationRequestModel(requestInfo);
-        return inputBoundary.createGameResponse(request);
+        GameCreationResponseModel response = inputBoundary.createGameResponse(request);
```

Comment:
whats the point of making this model only to not return anything except a bool? depending on where its used, we could package information about the start of the game (e.g. who the players are) and send it back to the game display

Reply:
That's probably a better idea. I just needed a way to get whether a Game had been made and thought that since the response only contains a boolean, that making a whole class to package it to a UI class is unnecessary.  To get the game information at first, would you have the GameCreationController interact with the GameController then?